framework/src/messaging/Kconfig

```diff
@@ -0,0 +1,32 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config MESSAGING_IPC
+	bool "Enable Messaging APIs"
+	default y
+	---help---
+		Enables Messaging APIs.
+
+if MESSAGING_IPC
+config MESSAGING_UNICAST_MAXMSG
+	int "Max message number for unicast"
```

Comment:
Is it needed?

Reply:
Yes. User can set the max number of messages